src/integrify/kapital/client.py

```diff
@@ -389,7 +443,7 @@ def partial_reverse_order(
                 amount: Ləğv olunacaq miqdar. Numerik dəyər.
             """  # noqa: E501
 
-        def create_order_for_pay_with_saved_card(
+        def order_with_saved_card(
```

Comment:
pay_with_saved_card: Ilishmeyim xirda gorune biler, amma EPointde de beledi, qalanlarinda da adlandirmani mumkun qeder yaxin saxlamaq meslehetlidi

Reply:
Bu son 3 funksiya KapitalClient-in icindeki pay_with_saved_card istifade edirem. Yeni bu funksiyalari sadece class daxilinde istifade olunur

Reply:
He tamam anladim